packages/cursorless-vscode-core/src/ide/vscode/VscodeOpenLink.ts

```diff
@@ -27,7 +27,7 @@ function getLinksForEditor(editor: vscode.TextEditor) {
   return vscode.commands.executeCommand(
     "vscode.executeLinkProvider",
     editor.document.uri,
-  ) as Thenable<vscode.DocumentLink[]>;
+  ) as Promise<vscode.DocumentLink[]>;
```

Comment:
Is this safe?

Reply:
Safe in what sense?  These links will only come from extensions that the user has installed.  If the user has a malicious extension, they're beyond our help; extensions have a full node context 😬